scripts/README.md

```diff
@@ -38,17 +39,35 @@ Anytime there are _new_ articles written, you need to run the script that genera
 1. Is this the first time that articles have been written for this section in this language?
 If so, then do both of the following:
 
-  * update the ["sections" config](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/scripts/generate_learning_path_markdown.js#L37) with the language code of the articles for the appropriate section.
+  * update the ["sections" config](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/scripts/section_data.json) with the language code of the articles for the appropriate section.
 Open a pull request for the change.
   * update the [Learning Path landing page](https://github.com/InnerSourceCommons/innersourcecommons.org/blob/master/resources/learningpath/index.md) with a link to your new language pages.
 
-3. Run **generate_learning_path_markdown.js** as described above.
+3. Run **generate_old_site_learning_path_markdown.js** as described above.
 3. `cp -r learningpath/* <path-to-innersourcecommons.org-repo>/resources/learningpath/`.
 3. Open a pull request with the modified files in the [InnerSourceCommons/innersourcecommons.org] repo.
 
 Note that these steps only needs to happen when there are new articles written.
 Changes to existing articles will automatically show up on the [innersourcecommons.org] site.
 
-[innersourcecommons.org]: http://www.innersourcecommons.org/
+## How to update innersourcecommons.net
+
+Anytime there are _any changes_, you need to run the script that generates the updated pages on the [innersourcecommons.net] site.
```

Comment:
This works, but just curious why it didn't work out to pull the article text dynamically from the _GitHub_ static content servers?

Reply:
I followed the implementation proposed here - https://github.com/InnerSourceCommons/innersourcecommons.net/issues/28 - which does it at build time, rather than run time, and it didn't occur to me to challenge it. This does make #349 a lot more important, but I think that's OK as we wanted to do this anyway.

I think this is probably better for a few reasons:
- Less custom JavaScript added to innersourcecommons.net, making development easier for future contributors to innersourcecommons.net
- More scaleable - don't need to rely on raw GitHub, which isn't really designed for static hosting as it isn't CDN fronted - see #248
- Marginally more performant for the user - don't need to wait for the additional network request to raw GItHub

Only downside is the need to complete #349, preferably before innersourceommons.net goes live!